babble/src/main/java/io/mosaicnetworks/babble/servicediscovery/mdns/MdnsAdvertiser.java

```diff
@@ -70,11 +74,9 @@ public MdnsAdvertiser(GroupDescriptor groupDescriptor, Context context) {
     }
 
     @Override
-    public boolean advertise(final String genesisPeers, final String currentPeers, PeersProvider peersProvider) {
-
-        mCurrentPeers = currentPeers;
+    public boolean advertise(BabbleNode node) {
```

Comment:
Does an MdnsAdvertiser need an entire BabbleNode, with all it's functionality?

Reply:
The MdnsAdvertiser only needs two methods to get the peers. But the WebRTC advertiser needs a bit more. It's a bit early to define an interface there.

Reply:
Ok, makes sense